roles/cfg_openwrt/templates/common/config/network.j2

```diff
@@ -53,7 +53,13 @@ config interface '{{ name }}'
   {% if role == 'corerouter' and ipv6_prefix is defined %}
     {% if 'ipv6_subprefix' in network %}
       {% set subprefix = ipv6_prefix | ipsubnet('64', network['ipv6_subprefix']) %}
+      {% if name == 'dhcp' %}
+	option ip6addr '{{ subprefix | ipaddr(1) | ipaddr('address') }}/56'
```

Comment:
i guess that will only work if the dhcp network is the first subprefix??

Reply:
```
{% for network in networks %}
  {% if role == 'corerouter' and ipv6_prefix is defined %}
    {% if 'ipv6_subprefix' in network %}
      {% if name == 'dhcp' %}
```

^^^ trying to organize my brain about which conditions have to be true for ipv6-pd to be enabled.

I dont think I understand how we are exposed to more than one subprefix. I think I must be reading this ansible wrong.

Reply:
Is `subpefix` a list? And `ipaddr(1)` selects the first subprefix in a list? And I guess `ipaddr(address)` returns the ip value?

Reply:
@spolack ping